Quantum-Control-Applications/ESR/T1.py

```diff
@@ -0,0 +1,165 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+################################
+# Open quantum machine manager #
+################################
+
+qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+
+########################
+# Open quantum machine #
+########################
+
+qm = qmm.open_qm(config)
+
+###################
+# The QUA program #
+###################
+
+pi_len = int(320//4)
+pi_half_len = int(160//4)
+
+wait_min = int(1500//4)
+wait_max = int(20000//4)
+dwait = 1000//4
+
+waits = np.arange(wait_min, wait_max + dwait/2, dwait)
```

Comment:
it's better to do it `+ 0.1` instead of `+ dwait/2`, otherwise it can give different results from QUA.
Not very critical as today we'll probably import the loop tools and then we can just use them.